### ISSUE TITLE
Docker image support for migration

### ISSUE THREAD

**Issue by mananpreetsingh** [zilliztech/vts#22]
Is there any Docker image that I can use to run this migration?

follow doc is empty: https://github.com/zilliztech/vts/blob/master/docs/zilliz/Docker%20Deployment.md

**Comment by nianliuu** [comment]
Yeah, sure. I'll release a docker image and update this doc before next week. Will notice you once it's done

**Comment by nianliuu** [comment]
you can also download the binary package in release page and follow the quick start guide: https://github.com/zilliztech/vts?tab=readme-ov-file#quickstart-guide

**Comment by nianliuu** [comment]
check out the latest dock, docker pull zilliz/vector-transport-service:latest for image

**Comment by mananpreetsingh** [comment]
thank you
